test/Kestrel.Core.Tests/Http2ConnectionTests.cs

```diff
@@ -29,16 +28,14 @@ public class Http2ConnectionTests : IDisposable, IHttpHeadersHandler
         {
             new KeyValuePair<string, string>(":method", "POST"),
             new KeyValuePair<string, string>(":path", "/"),
-            new KeyValuePair<string, string>(":authority", "127.0.0.1"),
-            new KeyValuePair<string, string>(":scheme", "https"),
+            new KeyValuePair<string, string>(":scheme", "http"),
```

Comment:
Why did these change?

Reply:
No strong reason. I just changed those to http because those tests are not using https.